### PR TITLE
Reorganize the user guide and update the get_started section

### PR DESCRIPTION
**Pull request by lvhan028**

### CODE REVIEW

File: docs/en/get_started.md [hunk 1]
```diff
@@ -1,69 +1,201 @@
...
+                    enable_prefix_caching=True,
+                    cache_max_entry_count=0.8,
+                    session_len=8192,
+                ))
+```
+
+```{note}
+The parameter "cache_max_entry_count" significantly influences the GPU memory usage. It means the proportion of FREE GPU memory occupied by the K/V cache after the model weights are loaded.
+The default value is 0.8. Once allocated, the K/V cache memory is reused repeatedly, which is why it is common to observe that the built pipeline and the api_server mentioned later in the next consumes a substantial amount of GPU memory.
+If you encounter an Out-of-Memory(OOM) error, you may need to consider lowering the value of cache_max_entry_count“.
```
Comment (RunningLeon) [inline docs/en/get_started.md:53]: lowering the value of `"cache_max_entry_count"`

File: docs/en/multi_modal/internvl.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# InternVL
```
Comment (RunningLeon) [inline docs/en/multi_modal/internvl.md:1]: this file is empty. Will it be updated in another PR?

Comment (lvhan028) [reply docs/en/multi_modal/internvl.md:1]: Yes, in another PR. I will mark TODO in it

File: docs/zh_cn/installation.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
+# 安装
```
Comment (AllentDan) [inline docs/zh_cn/installation.md:1]: what's the plan for `build.md`?

**Review by RunningLeon — APPROVED** [review]
LGTM

File: docs/zh_cn/installation.md (hunk 3 above)
Comment (lvhan028) [reply docs/zh_cn/installation.md:1]: I forgot to remove it

Comment (lvhan028) [reply docs/zh_cn/installation.md:1]: I removed it.

**Review by AllentDan — APPROVED** [review]
